nuitka/tools/gen_helper_code/BINARY_ADD.py

```diff
@@ -0,0 +1,33 @@
+
+
+import jinja2
+
+"""
+def BINARY_ADDITION(a, b):
+	if(isinstance(a,str)):
+		print("Not an integer") 
+	else:
+		if (type(int(a)))==int:
+			if(isinstance(a,float)):
+				print("Not an Integer")
+			else:
+				c = a+b
+				return c
+"""
+macro = ("{% macro add(a,b) %}"
+			"{# if PYTHON_VERSION < 300 #}"
+         		"{% if a is number %}"
+         			"{% if b|int !=0 %}"
+         				"{{ a+b }}"
+         			"{% endif %}"
+         		"{% endif %}"
+         	"{# endif #}"
+        "{% endmacro %}")
+use_macro = "Add a + b = {{ add(a=2,b=5)}}"
+loader = jinja2.DictLoader({'template': use_macro})
+env = jinja2.Environment(loader=loader)
+macro_template = env.from_string(macro)
+env.globals['add'] = macro_template.module.add
```

Comment:
So I see you figured out how to use an environment, but I still don't think we need one really. Recall my comments about how methods of type description objects can basically do all we want already. There is barely a need for global stuff. And if, indeed, put python_version as a global variable like that.

Reply:
Environment was necessary to call macro that's why that was needed to be implemented.